examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,286 @@
+MUTAG dataset. We'll start by loading these sample sets and visualizing the corresponding graphs.
+"""
+
+from strawberryfields.gbs import data, plot, similarity
+
+m0 = data.Mutag0()
+m1 = data.Mutag1()
+m2 = data.Mutag2()
+m3 = data.Mutag3()
+
+##############################################################################
+# These datasets contain both the adjacency matrix of the graph and the samples generated through
+# GBS. We can access the adjacency matrix through:
+
+m0_a = m0.adj
+m1_a = m1.adj
+m2_a = m2.adj
+m3_a = m3.adj
+
+##############################################################################
+# Samples from these graphs can be accessed by indexing:
+
+print(m0[0])
+
+##############################################################################
+# We can now plot the four graphs using the :mod:`~.gbs.plot` module. To use this module,
+# we need to convert the adjacency matrices into NetworkX Graphs:
+
+import networkx as nx
+import plotly
+
+plot_mutag_0 = plot.plot_graph(nx.Graph(m0_a))
+plot_mutag_1 = plot.plot_graph(nx.Graph(m1_a))
+plot_mutag_2 = plot.plot_graph(nx.Graph(m2_a))
+plot_mutag_3 = plot.plot_graph(nx.Graph(m3_a))
+
+plotly.offline.plot(plot_mutag_0, filename="MUTAG_0.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_0.html
+
+plotly.offline.plot(plot_mutag_1, filename="MUTAG_1.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_1.html
+
+plotly.offline.plot(plot_mutag_2, filename="MUTAG_2.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_2.html
+
+plotly.offline.plot(plot_mutag_3, filename="MUTAG_3.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_3.html
+#
+# By visual inspection, we see that the graphs of ``m1_a`` and ``m2_a`` look very similar. In fact,
+# it turns out that they are *isomorphic* to each other, which means that the graphs can be made
+# identical by permuting their node labels.
+
+##############################################################################
+# Creating a feature vector
+# -------------------------
+#
+# Following :cite:`schuld2019quantum`, we can create a *feature vector* to describe each graph.
+# These feature vectors contain information about the graphs and can be viewed as a mapping to a
+# high-dimensional feature space, a technique often used in machine learning that allows us to
+# employ properties of the feature space to separate and classify the vectors.
+#
+# The feature vector of a graph can be composed in a variety of ways. One approach is to build it
+# using statistics from a GBS device configured to sample from the graph, as we now discuss.

```

Comment:
I would give a 1-sentence explanation here, so the reader has a motivation and guidance when reading on. For example: "One example is to associate features with the relative frequencies of certain types of measurements being recorded from a GBS device configured ..."

Reply:
Thanks, have added this!